### DISCUSSION TITLE
Where can I find programs that integrate with Nushell well?

### DISCUSSION THREAD

**Discussion by dch82** [nushell/nushell#9753] [Q&A]
That is, designed around Nushell and inputting and outputting data that can be directly processed by Nushell.

I find the idea of Nushell excellent, but there are basically no third-party programs that are integrated with Nushell.

Thanks!
Danche

**Comment by amtoine** [comment]
@dch82 
maybe you can find a list of some projects in https://github.com/nushell/awesome-nu :yum:
